README.md

```diff
@@ -15,6 +15,114 @@ bin/cake plugin load Robotusers/Tactician
 
 ## Using the plugin
 
+### CakePHP integration
+
+This plugin provides Controller and Model integration through [Commander](https://github.com/robotusers/commander) library.
+
+Commander is a command bus abstraction library for PHP which enables you to decouple your code from a concrete command bus implementation.
+
+#### Using the Commander (PHP 7.1+)
+
+Install `robotusers/commander`:
+
+```
+composer require robotusers/commander
+```
+
+Set up your controllers:
+
+```php
+use Cake\Controller\Controller;
+use Robotusers\Commander\CommandBusAwareInterface;
+use Robotusers\Commander\CommandBusAwareTrait;
+
+class OrdersController extends Controller implements CommandBusAwareInterface
+{
+   use CommandBusAwareTrait;
+
+   public function makeOrder()
+   {
+        // ...
+        $command = new MakeOrderCommand($data);
+        $this->handleCommand($command);
```

Comment:
Why is this a trait (with potential for clashes) instead of a Component or just a (n invokable?) class?

Reply:
The trait implements the interface and adds a helper method for handling commands.